filters/apiusagemonitoring/spec_test.go

```diff
@@ -222,7 +222,8 @@ func Test_CreateFilter_FullConfigSingleApi(t *testing.T) {
 		"path_templates": [
```

Comment:
We can simplify the path_template, since we now test the generation of labels and regular expressions in its own unit test.

Reply:
Done.

Reply:
Done.